packages/cli/package.json

```diff
@@ -10,8 +10,9 @@
     "@oclif/command": "^1.5.19",
```

Comment:
Please use exact versions of your dependencies for better reproducibility.

Reply:
I think the `package-lock.json` is enough, since we use `npm ci` to install dependencies, so the builds are pretty reproducible.